rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -423,7 +426,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 # This rule has two stricter sibling: 942361 and 942362.
 # The keywords 'alter' and 'union' led to false positives.
 # Therefore they have been moved to PL2 and the keywords have been extended on PL1.
-# The original version also had loose word boundaries and context checksm cause further false positives. 
+# The original version also had loose word boundaries and context checksm cause further false positives.
```

Comment:
```suggestion
# The original version also had loose word boundaries and context checksum cause further false positives.
```

Just as a clean core move, it may as well be spelt correctly. Unless it was done purposefully for line length reasons?